docs/language/contracts.mdx

```diff
@@ -387,6 +387,26 @@ let signer: AuthAccount = ...
 let contract = signer.contracts.get(name: "Test")
 ```
 
+### Borrowing a Deployed Contract
+
+A reference to a deployed contract contract can obtained using the `borrow` function:
+
+  ```cadence
+  fun borrow<T: &Any>(name: String): T?
+  ```
+
+  This returns a reference to the contract value stored with that name on the account, if it exists, and if it has
+  the provided type `T`. 
+
+  Returns `nil` if no contract/contract interface with the given name exists in the account.
+
+For example, assuming that a contract named `Test` is deployed to an account with type `TestContract`, the contract can be retrieved as follows:
+
+```cadence
+let signer: AuthAccount = ...
+let contract: &TestContract = signer.contracts.borrow<&TestContract>(name: "Test")
+```
```

Comment:
Might be more clear to emphasize that `TestContract` is actually an interface to which the `Test` contract conforms to. 
```suggestion
For example, assuming that a contract named `Test` which conforms to the `TestInterface` interface is deployed to an account, the contract can be retrieved as follows:

```cadence
let signer: AuthAccount = ...
let contract: &TestInterface = signer.contracts.borrow<&TestInterface>(name: "Test")
```
```